python/ray/tune/trainable/trainable.py

```diff
@@ -1131,7 +1131,7 @@ def cleanup(self):
         If any Ray actors are launched in the Trainable (i.e., with a RLlib
         trainer), be sure to kill the Ray actor process here.
 
-        You can kill a Ray actor by calling `actor.__ray_terminate__.remote()`
+        You can kill a Ray actor by calling `ray.kill(actor)`
```

Comment:
can we also mention that this method is supposed to be light-weight. Also the forced clean up will kick in after a default of 600 seconds (maybe pointing them to the env doc string).